docs/postgresql/topics/server_auth.md

```diff
@@ -0,0 +1,41 @@
+To start PostgreSQL in SSL mode, first enable SSL in `postgresql.conf`.
+
+```bash
+# ...
+ssl = on
+# ...
+```
+
+Put your `{{ server_cert }}` and `{{ server_key }}` files in your installation's data directory, often at `/var/lib/pgsql/data` or `/usr/local/pgsql/data`. Make sure their filenames are `server.crt` and `server.key` respectively, which are the expected defaults.
+
+```shell-session
+$ sudo cp {{ server_cert }} /var/lib/pgsql/data/server.crt
+$ sudo cp {{ server_key }} /var/lib/pgsql/data/server.key
+```
+
+You'll need to maually set the file permissions to disallow access to world or group.
+
+```shell-session
+$ sudo chown postgres:postgres /var/lib/pgsql/data/server.{crt,key}
+$ sudo chmod 0600 /var/lib/pgsql/data/server.{crt,key}
+```
+
+If you'd like to specify a different path for these files, manually configure them in `postgresql.conf`.
+
+```bash
+# ...
+ssl_cert_file = '/path/to/server.crt'
+ssl_key_file = '/path/to/server.key'
+# ...
+```
+
+In your `pg_hba.conf` file, change all records for non-local connections from `host` to `hostssl` to require clients to connect over TLS. It might look something like this.
+
+``` bash
+# TYPE  DATABASE        USER            ADDRESS                 METHOD
+
+# ...
+hostssl all             all             all                     md5
```

Comment:
Method md5? or cert?

Reply:
Since this section covers server auth, it's just `md5` with `hostssl`. In the client auth section, it covers `clientcert=1` or even passwordless `cert`.

Reply:
Sounds good.